src/youwol/app/environment/models/models_features.py

```diff
@@ -0,0 +1,7 @@
+# standard library
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True, kw_only=True)
```

Comment:
We usually use `BaseModel` as base class to define configuration's related models, is using `dataclass` intended?